zipkin-lens/package.json

```diff
@@ -88,6 +88,7 @@
     "react-virtualized": "^9.21.1",
     "redux": "^4.0.0",
     "redux-thunk": "^2.3.0",
+    "regenerator-runtime": "^0.13.2",
```

Comment:
We need to install core-js & regenerator-runtime for using polyfill.

Please see: 
* https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md#usebuiltins-usage-with-corejs-3